src/plugins/volumeBooster/index.ts

```diff
@@ -77,6 +77,11 @@ export default definePlugin({
                     match: /Math\.max.{0,30}\)\)/,
                     replace: "arguments[0]"
                 },
+                // Fix streams not playing audio untill you update them
+                {
+                    match: /(?=}return"video")/,
```

Comment:
matching nothing is confusing and likely worse for performance
```suggestion
                    match: /\}return"video"/,
```